app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -37,7 +37,7 @@ class CasesService @Inject()(appConfig: AppConfig, auditService: AuditService,
   private def addEvent(original: Case, updated: Case, operator: Operator, comment: Option[String] = None)
                       (implicit hc: HeaderCarrier): Future[Unit] = {
     val event = NewEventRequest(CaseStatusChange(original.status, updated.status, comment), operator)
-    connector.createEvent(updated, event)
+    connector.createEvent(updated.reference, event)
```

Comment:
As mentioned above, in my opinion it was better as it was

Reply:
Fixed